packages/x-components/src/components/animations/animate-clip-path/animate-clip-path.style.scss

```diff
@@ -63,7 +63,7 @@
   }
 
   &--right-to-left {
-    &#{$p}--enter {
+    &#{$p}--enter-from {
```

Comment:
There are many more of:

- Replace instances of .v-enter to .v-enter-from
- Replace instances of .v-leave to .v-leave-from

Along the animations. Do you plan to refactor them in next PRs?

Reply:
I just fixed this in the "animations factory" related files. I will check the rest.

Reply:
In future PRs is ok. This comment was more a reminder than a request 😄